docs/source/contributors/structure.rst

```diff
@@ -19,6 +20,27 @@ Build (``build``)
 This directory contains scripts to allow for installing packages such as Mantid
 through setuptools.
 
+*******************
+Docker (``Docker``)
```

Comment:
We should probably put the docker stuff into the [install instructions](https://fitbenchmarking.readthedocs.io/en/latest/users/install_instructions/index.html) as well.

Reply:
I was just typing this up, but I don't think the Docker stuff is useful to a user yet.  In particular:
- fitbenchmarking isn't included, so we'd have to explain some gymnastics where the user pulls locally or shares the filesystem, and then does pip install
- the docker containers don't include a browser, so nothing useful is output

I think we should have an issue to make further containers which contain fitbenchmarking + dependencies, but I don't think we should complicate the install instructions quite yet.

Reply:
Created #614 to address this